lib/capistrano/tasks/laravel.rake

```diff
@@ -16,9 +16,17 @@ namespace :laravel do
       end
     end
   end
+
+  desc 'Restart queue'
+  task :restart_queue_worker, :on_error => :continue do
+    on roles(:all) do
+      run "ps -ef | grep 'queue:work' | awk '{print $2}' | xargs sudo kill -9"
```

Comment:
Not super important, but can we do this with [`php artisan queue:restart`](https://laravel.com/docs/5.5/queues#queue-workers-and-deployment) so we don't kill any in-process jobs?